first_5_steps/1_install_tt_buda.md

```diff
@@ -58,6 +58,7 @@ Please navigate to [tt-kmd](https://github.com/tenstorrent/tt-kmd).
 ### Device Firmware Update
 
 Please navigate to [tt-flash](https://github.com/tenstorrent/tt-flash) and [tt-firmware-gs](https://github.com/tenstorrent/tt-firmware-gs).
```

Comment:
Remove this line. It's redundant with the next one.

Reply:
will do this

Reply:
done